src/system/data-import-price-tier.md

```diff
@@ -4,90 +4,84 @@ title: Importing Tier Prices
 
-1.  Under _Export Settings_, set **Entity Type** to `Advanced Pricing`.
+1. Under _Export Settings_, set **Entity Type** to `Advanced Pricing`.
 
-1.  In the **Entity Attributes** grid, scroll down to the SKU attributes. Then, do the following:
+1. In the **Entity Attributes** grid, scroll down to the SKU attributes and do the following:
 
-    -  For tier prices that are based on a discount percentage. enter the SKU of each product to be exported, separated by a comma.
+    - For tier prices that are based on a discount percentage, enter the SKU of each product to be exported, separated by a comma.
 
-        ![]({% link images/images/price-tier-export-sku.png %}){: .zoom}
+        ![Data export - product SKUs]({% link images/images/price-tier-export-sku.png %}){: .zoom}
         _SKU(s) to Export_
 
-    -  For tier prices that are based on a fixed amount. enter the SKU of each product.
+    - For tier prices that are based on a fixed amount, enter the SKU of each product.
 
-    -  Scroll down and click **Continue**.
+    - Scroll down and click **Continue**.
 
-1.  Look in the lower-left corner of your browser window for the export file. Then, open the file.
+1. Locate the export file at the downloads location for your web browser and open the file.
 
-    <!--{% if "Default.CE Only,Default.EE Screenshot" contains site.edition %}-->
-
-    ![]({% link images/images/price-tier-customer-group-discount-export.png %}){: .zoom}
+    <!--{% if "Default.CE Only,Default.EE Only" contains site.edition %}-->
+    ![Example - exported tier price data]({% link images/images/price-tier-customer-group-discount-export.png %}){: .zoom}
     <!--{% endif %}-->
-
     <!--{% if "Default.B2B Only" contains site.edition %}-->
-
-    ![]({% link images/images-b2b/price-tier-shared-catalog-export-data.png %}){: .zoom}
+    ![Example - exported tier price data]({% link images/images-b2b/price-tier-shared-catalog-export-data.png %}){: .zoom}
     <!--{% endif %}-->
+    _Exported Tier Price Data_
 
 ### Exported Tier Price Data
 
 The following columns are included in the exported data:
 
--   `sku`
--   `tier_price_website`
--   `tier_price_customer_group`
--   `tier_price_qty`
--   `tier_price`
--   `tier_price_value_type`
+- `sku`
+- `tier_price_website`
+- `tier_price_customer_group`
+- `tier_price_qty`
+- `tier_price`
+- `tier_price_value_type`
 
 You use the exported data as a template for importing tier price data.
 
 ## Step 2: Update the Data
 
-1.  Update the tier price data for each product, as needed.
+1. Update the tier price data for each product, as needed.
 
-1.  Any products without tier price updates can be deleted from the CSV file. There is no need to reimport products that haven’t changed.
+    Any products without tier price updates can be deleted from the CSV file. There is no need to reimport products that have not changed.
 
-1.  **Save** the updated CSV file.
+1. **Save** the updated CSV file.
 
     {:.bs-callout-info}
```

Comment:
i disagree with this change.  when looking at a file size, it is a number, not text.